src/widgets/Menu/Menu.tsx

```diff
@@ -67,7 +69,9 @@ const Menu: React.FC<NavProps> = ({
   links,
   children,
 }) => {
-  const [isPushed, setIsPushed] = useState(true);
+  const { isXl } = useMatchBreakpoints();
+  const isMobile = isXl === false;
```

Comment:
This seems weird but the current behavior of the menu treats `lg` as "mobile" with regards to style.